addons/portal/static/src/xml/portal_chatter.xml

```diff
@@ -45,7 +45,7 @@
                                         <div class="d-flex px-1">
                                             <button class="o_portal_chatter_attachment_btn btn fa fa-paperclip border-0" type="button" title="Add attachment"/>
                                         </div>
-                                        <button t-attf-data-action="/mail/chatter_post" class="o_portal_chatter_composer_btn btn btn-primary o-last rounded-3" type="submit">Send</button>
+                                        <button t-attf-data-action="/mail/chatter_post" class="o_portal_chatter_composer_btn btn btn-primary o-last rounded-3" type="submit"/>
```

Comment:
We need a default button name when `portal_rating` is not installed so we should probably keep this text here :slightly_smiling_face: